crates/host/src/wasmbus/mod.rs

```diff
@@ -335,93 +334,106 @@ fn decode_empty_provider_response(buf: impl AsRef<[u8]>) -> anyhow::Result<()> {
 impl Blobstore for Handler {
     #[instrument]
     async fn create_container(&self, name: &str) -> anyhow::Result<()> {
-        let target = self
-            .identify_interface_target(&CallTargetInterface::from_parts((
+        use wrpc_interface_blobstore::Blobstore;
+
+        let WrpcInterfaceTarget { id, .. } = self
+            .identify_wrpc_target(&CallTargetInterface::from_parts((
                 "wasi",
                 "blobstore",
                 "blobstore",
                 None,
             )))
-            .await?;
-        self.call_operation(
-            target,
-            "wasmcloud:blobstore/Blobstore.CreateContainer",
-            &name,
-        )
-        .await
-        .and_then(decode_empty_provider_response)
+            .await?
+            .context("unknown target")?;
+        let wrpc =
+            wrpc_transport_nats::Client::new(self.nats.clone(), format!("{}.{id}", self.lattice));
```

Comment:
Note to myself, need to ensure we are propagating traces for each interface here too.